scripts/postcompile.mjs

```diff
@@ -10,23 +10,14 @@
 
 import path from 'path';
 import { writeFileSync } from 'fs';
-import { rimraf } from 'rimraf';
```

Comment:
We can remove `rimraf` from top-level dev deps now and update lockfile.

Reply:
And `glob` it looks like

Reply:
Done